Stanford/JumpstartEngineering/Install/Courses/EnableCourses.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @file
+ * Enable Courses
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Courses;
+
+/**
+ *
+ */
+class EnableCourses extends \ITasks\AbstractInstallTask {
+
+  /**
+   * Enable Courses on JSE sites
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
This function requires that redirect, features, and path be enable. Please check for them throw an error if they don't exist.